scripts/create_datacheck_tickets.pl

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env perl
+
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+=head2 DESCRIPTION
+
+This script parses the output from Ensembl's Datachecks and creates JIRA tickets
+for each failure.
+
+=head1 SYNOPSIS
+
+perl create_datacheck_tickets.pl [options]
```

Comment:
```suggestion
perl create_jira_tickets.pl [options]
```
I think there's less potential for confusion if we have 'jira' in the script name; it makes it clearer what the script does, and distinguishes it from the existing `create_datacheck.pl` script.